app/Core/dbconfig.ini

```diff
@@ -0,0 +1,5 @@
+[data]
```

Comment:
Call this file `serverconfig.ini`. Also make this section header `database`.

Reply:
Also Move this file to `app/`

Reply:
done, but what should the new filepath be?